x-pack/plugins/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -64,7 +64,7 @@ export const ruleRegistrySearchStrategyProvider = (
 
       const registeredRuleTypes = alerting.listTypes();
 
-      const [validRuleTypeIds, invalidRuleTypeIds] = partition(request.ruleTypeIds, (ruleTypeId) =>
+      const [validRuleTypeIds] = partition(request.ruleTypeIds, (ruleTypeId) =>
```

Comment:
I didn't try it but if `partition` always returns two elements maybe it is better to do `const [validRuleTypeIds, _] =`